src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -156,12 +157,23 @@ public void setGenotypingErrorRate(final double genotypingErrorRate) {
     }
 
     /**
-     * Sets the maximal difference in PL scores considered when reading PLs from a VCF.
+     * Does same thing as {@link #setMaximalPLDifference(int)} but named in the compliant way.
+     *
+     * @deprecated use {@link #setMaximalPLDifference(int)} instead.
```

Comment:
Since the method you're forwarding to is also deprecated, it seems like this comment should also mention `CappedHaplotypeProbabilities`